menu.go

```diff
@@ -15,8 +15,9 @@ func NewMenu(label string, items ...*MenuItem) *Menu {
 // MenuItem is a single item within any menu, it contains a display Label and Action function that is called when tapped.
 type MenuItem struct {
 	ChildMenu   *Menu
-	IsSeparator bool
```

Comment:
Is the re-ordering required/correct here?

Reply:
It optimizes the struct for memory alignment having the two bool values after each other. `Label` seemed more central to a menu item than the 2 boolean values, so I decided to move `Label` before `IsSeparator`.

If that is not something you care about (memory alignment), I can undo the move.

Reply:
I moved the `IsQuit` before `Label` now, undoing the reorder but keeping the two boolean values together.